models/ed/tests/testthat/test.write.configs.ed.R

```diff
@@ -21,6 +21,7 @@ testdir <- tempfile()
 dir.create(testdir)
 withr::defer(unlink(testdir, recursive = TRUE))
 unzip("data/outdir.zip", exdir = testdir)
+# unzip("models/ed/tests/testthat/data/outdir.zip", exdir = testdir)
```

Comment:
when is this used? for interactive debugging?

Reply:
Exactly.  Tests are run with `testthat/` as the working directory rather than the project root.